.travis.yml

```diff
@@ -16,21 +16,44 @@ matrix:
     - os: linux
       dist: trusty
       sudo: required
+      env:
+        - MYSQL_VER=5
+    - os: linux
+      dist: trusty
+      sudo: required
+      env:
+        - MYSQL_VER=8
+    - os: osx
+      osx_image: xcode9.2
+      sudo: required
+      env:
+        - SWIFT_SNAPSHOT=4.0.3 MYSQL_VER=5
     - os: osx
       osx_image: xcode9.2
       sudo: required
-      env: SWIFT_SNAPSHOT=4.0.3
+      env:
+        - SWIFT_SNAPSHOT=4.0.3 MYSQL_VER=8
     - os: osx
       osx_image: xcode9.4
       sudo: required
     - os: linux
       dist: trusty
       sudo: required
-      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-07-08-a
+      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-07-08-a MYSQL_VER=5
+    - os: linux
+      dist: trusty
+      sudo: required
+      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-07-08-a MYSQL_VER=8
+    - os: osx
+      osx_image: xcode9.4
+      sudo: required
+      env:
+        - SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-06-04-a MYSQL_VER=5
```

Comment:
Now that Travis has an `xcode10` image I think we can move up to a newer 4.2 snapshot